.github/workflows/functional-test-cloud.yaml

```diff
@@ -549,7 +549,17 @@ jobs:
       - uses: azure/setup-helm@v4
         with:
           version: ${{ env.HELM_VER }}
+      
+      # The role-to-assume is the role that the github action will assume to execute aws commands and 
```

Comment:
did you run this workflow to test it? if so, could you link the run in the PR comments?

Reply:
https://github.com/radius-project/radius/actions/runs/12207284546

Reply:
do we need this in long-running tests?